server/tserver/src/main/java/org/apache/accumulo/tserver/ScanServer.java

```diff
@@ -881,7 +903,7 @@ public InitialMultiScan startMultiScan(TInfo tinfo, TCredentials credentials,
       batch.put(extent, entry.getValue());
     }
 
-    try (ScanReservation reservation = reserveFiles(batch.keySet())) {
+    try (ScanReservation reservation = reserveFiles(batch)) {
 
       HashMap<KeyExtent,TabletBase> tablets = new HashMap<>();
       batch.keySet().forEach(extent -> {
```

Comment:
The following should be the set of non failure tablets.

```suggestion
      reservation.tabletMetadata.keySet().forEach(extent -> {
```

Reply:
Addressed in 381dac3